apps/pie-docs/src/patterns/haptic-feedback/overview/overview.md

```diff
@@ -0,0 +1,66 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: 'Haptic Feedback'
+    order: 1
+shouldShowContents: true
+permalink: patterns/haptic-feedback/
+---
+
+
+## What are haptics?
+
+Haptics refer to the incorporation of tactile feedback into user interfaces to enhance the overall user experience. It involves the use of touch-based sensations, such as vibrations or pulses, in response to user interactions with our products.
+
+By integrating haptic feedback, we aim to provide users with a more intuitive and engaging interaction, making the digital experience feel more tangible and responsive.
+
+{% contentPageImage {
+  src:"../../../assets/img/patterns/haptic-feedback/overview-desktop.svg",
+  mobileSrc: "../../../assets/img/patterns/haptic-feedback/overview-mobile.svg",
+  alt: "Visual representation of haptic feedback."
```

Comment:
I wonder if we can be a bit more descriptive about this image - e.g. "Illustration of a mobile device surrounded by shapes that indicate vibration"